openreview/journal/recruitment.py

```diff
@@ -48,7 +48,12 @@ def invite_action_editors(self, message, subject, invitees, invitee_names=None):
                     recruitment_status['invited'].append(invitee)
                     assert self.client.get_groups(id=action_editors_invited_id, member=invitee)
                 except Exception as e:
-                    self.client.remove_members_from_group(action_editors_invited_id, invitee)
+                    try:
```

Comment:
if remove members from group is causing the expecting then I would change the implementation of remove_members_from_group and check if the group exists before posting the group edit asking to remove the groups.